env_test.go

```diff
@@ -403,7 +403,7 @@ func TestParsesEnv(t *testing.T) {
 
 func TestParsesEnv_Map(t *testing.T) {
 	type config struct {
-		MapStringString map[string]string `env:"MAP_STRING_STRING" envSeparator:","`
+		MapStringString map[string]string `env:"MAP_STRING_STRING" envSeparator:","  envKeyValSeparator:"|"`
```

Comment:
This effectively stops testing for the default separator (unless there's another test that does it).

I think we can leave the previous one, and add another line using another env and separator, something like:

```go
MapStringString                map[string]string `env:"MAP_STRING_STRING" envSeparator:","`
CustomSeparatorMapStringString map[string]string `env:"CUSTOM_MAP_STRING_STRING" envSeparator:","  envKeyValSeparator:"|"`
```

Reply:
Fixed